AutomationTests/Malafi.Tests/Features/DocumentsTypeDetails.feature

```diff
@@ -0,0 +1,60 @@
+﻿Feature: DocumentsTypeDetails
+
+A short summary of the feature
+Background: Login to Internal Portal and click on Malafi Link
```

Comment:
Please change the description to be:
Common steps till reach to add document type page